.github/workflows/docker_publish.yml

```diff
@@ -184,196 +109,42 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      - name: Build and push Dockerfile_3_moab
-        uses: docker/build-push-action@v2
+      - name: Test DAGMC in Docker image
+        uses: docker/build-push-action@v3
         with:
           file: CI/Dockerfile
-          target: moab
+          target: dagmc_test
           context: .
```

Comment:
I'm thinking we may not need this line in this case. The [action documentation](https://github.com/docker/build-push-action#git-context) indicates that the right version of the repository will be checked out in the Docker image by default. This isn't true in the case of our dependencies though, so I can see why we'd need it there.

Reply:
this line has been removed from the yml file so I think this one is resolved